docs/pages/kubernetes-access/controls.mdx

```diff
@@ -445,6 +445,13 @@ value begins with `^` and ends in `$`, the Kubernetes Service will treat it as a
 regular expression using Go's `re2` syntax (see the `re2`
 [README](https://github.com/google/re2/wiki/Syntax)).
 
+<Notice type="tip" >
+  For a user to have Kubernetes access `kubernetes_groups` or `kubernetes_users`
+  must also be assigned within a role to a user. `kubernetes_resources` configuration
```

Comment:
I read this a few times and the wording feels off. Maybe something like this instead?

> For a user to access Kubernetes clusters, they must be assigned a role
> that contains `kubernetes_groups` or `kubernetes_users`.

Reply:
please see update